dtl/gifs.py

```diff
@@ -19,4 +19,5 @@
 hayasaka = "https://i.imgur.com/9QamQDE.gif"
 very_nice = "https://tenor.com/sbvk.gif"
 nice_to_meet_you = "http://gph.is/2ebqM3y"
-cheesesteak = "https://www.mcall.com/resizer/kOnDj8vjKooAtTmKZwvUI-m01t8=/800x673/top/arc-anglerfish-arc2-prod-tronc.s3.amazonaws.com/public/5TEPU2PMSNBGXDC2TA6JFVD6KQ.jpg"
\ No newline at end of file
+cheesesteak_joe = "https://www.mcall.com/resizer/kOnDj8vjKooAtTmKZwvUI-m01t8=/800x673/top/arc-anglerfish-arc2-prod-tronc.s3.amazonaws.com/public/5TEPU2PMSNBGXDC2TA6JFVD6KQ.jpg"
```

Comment:
Although this is a personal repo, we still need to enforce proper naming standards. I would recommend renaming the variables to `cheesesteak_sleepy_joe` and `cheesesteak_maga2020!`.